src/Bot.mjs

```diff
@@ -487,8 +486,7 @@ class SlackBot extends Adapter {
     // Ignore messages we've already seen
     if (this.seenMessages.has(message.body.event.client_msg_id)) {
       this.robot.logger.debug(`Ignoring message ${message.body.event.client_msg_id}`);
-      if (!message?.ack) return;
-      return await message.ack();
+      return;
     }
 
     this.seenMessages.add(message.body.event.client_msg_id);
```

Comment:
I left `this.seenMessages` in case there are messages with no `ack`?

Reply:
seenMessages actually tracks duplicate messages if the script is taking longer than Slack's message timeout.

Reply:
seenMessages dedupe makes sense for long running message handling. I'm sure this is working.

When I halt in a faster listener:
- I wait long enough for the socket to resend the msg (I think 3 seconds)
- Let the code continue
- The end of the current eventHandler completes
    - The msg ID is removed from seenMessages
    - message.ack() is called
-  The new eventHandler starts so seenMessages is empty when this check is made